show_pdf_tags/decode.lua

```diff
@@ -65,7 +65,148 @@ local pdfdoc_to_utf8 = lpeg.Cs((
+  ['\x8D'] = '\u{008D}',
+  ['\x8E'] = '\u{017D}',
+  ['\x8F'] = '\u{008F}',
+  ['\x90'] = '\u{0090}',
+  ['\x91'] = '\u{2018}',
+  ['\x92'] = '\u{2019}',
+  ['\x93'] = '\u{201C}',
+  ['\x94'] = '\u{201D}',
+  ['\x95'] = '\u{2022}',
+  ['\x96'] = '\u{2013}',
+  ['\x97'] = '\u{2014}',
+  ['\x98'] = '\u{02DC}',
+  ['\x99'] = '\u{2122}',
+  ['\x9A'] = '\u{0161}',
+  ['\x9B'] = '\u{203A}',
+  ['\x9C'] = '\u{0153}',
+  ['\x9D'] = '\u{009D}',
+  ['\x9E'] = '\u{017E}',
+  ['\x9F'] = '\u{0178}',
+  ['\xA0'] = '\u{00A0}',
+  ['\xA1'] = '\u{00A1}',
+  ['\xA2'] = '\u{00A2}',
+  ['\xA3'] = '\u{00A3}',
+  ['\xA4'] = '\u{00A4}',
+  ['\xA5'] = '\u{00A5}',
+  ['\xA6'] = '\u{00A6}',
+  ['\xA7'] = '\u{00A7}',
+  ['\xA8'] = '\u{00A8}',
+  ['\xA9'] = '\u{00A9}',
+  ['\xAA'] = '\u{00AA}',
+  ['\xAB'] = '\u{00AB}',
+  ['\xAC'] = '\u{00AC}',
+  ['\xAD'] = '\u{00AD}',
+  ['\xAE'] = '\u{00AE}',
+  ['\xAF'] = '\u{00AF}',
+  ['\xB0'] = '\u{00B0}',
+  ['\xB1'] = '\u{00B1}',
+  ['\xB2'] = '\u{00B2}',
+  ['\xB3'] = '\u{00B3}',
+  ['\xB4'] = '\u{00B4}',
+  ['\xB5'] = '\u{00B5}',
+  ['\xB6'] = '\u{00B6}',
+  ['\xB7'] = '\u{00B7}',
+  ['\xB8'] = '\u{00B8}',
+  ['\xB9'] = '\u{00B9}',
+  ['\xBA'] = '\u{00BA}',
+  ['\xBB'] = '\u{00BB}',
+  ['\xBC'] = '\u{00BC}',
+  ['\xBD'] = '\u{00BD}',
+  ['\xBE'] = '\u{00BE}',
+  ['\xBF'] = '\u{00BF}',
+  ['\xC0'] = '\u{00C0}',
+  ['\xC1'] = '\u{00C1}',
+  ['\xC2'] = '\u{00C2}',
+  ['\xC3'] = '\u{00C3}',
+  ['\xC4'] = '\u{00C4}',
+  ['\xC5'] = '\u{00C5}',
+  ['\xC6'] = '\u{00C6}',
+  ['\xC7'] = '\u{00C7}',
+  ['\xC8'] = '\u{00C8}',
+  ['\xC9'] = '\u{00C9}',
+  ['\xCA'] = '\u{00CA}',
+  ['\xCB'] = '\u{00CB}',
+  ['\xCC'] = '\u{00CC}',
+  ['\xCD'] = '\u{00CD}',
+  ['\xCE'] = '\u{00CE}',
+  ['\xCF'] = '\u{00CF}',
+  ['\xD0'] = '\u{00D0}',
+  ['\xD1'] = '\u{00D1}',
+  ['\xD2'] = '\u{00D2}',
+  ['\xD3'] = '\u{00D3}',
+  ['\xD4'] = '\u{00D4}',
+  ['\xD5'] = '\u{00D5}',
+  ['\xD6'] = '\u{00D6}',
+  ['\xD7'] = '\u{00D7}',
+  ['\xD8'] = '\u{00D8}',
+  ['\xD9'] = '\u{00D9}',
+  ['\xDA'] = '\u{00DA}',
+  ['\xDB'] = '\u{00DB}',
+  ['\xDC'] = '\u{00DC}',
+  ['\xDD'] = '\u{00DD}',
+  ['\xDE'] = '\u{00DE}',
+  ['\xDF'] = '\u{00DF}',
+  ['\xE0'] = '\u{00E0}',
+  ['\xE1'] = '\u{00E1}',
+  ['\xE2'] = '\u{00E2}',
+  ['\xE3'] = '\u{00E3}',
+  ['\xE4'] = '\u{00E4}',
+  ['\xE5'] = '\u{00E5}',
+  ['\xE6'] = '\u{00E6}',
+  ['\xE7'] = '\u{00E7}',
+  ['\xE8'] = '\u{00E8}',
+  ['\xE9'] = '\u{00E9}',
+  ['\xEA'] = '\u{00EA}',
+  ['\xEB'] = '\u{00EB}',
+  ['\xEC'] = '\u{00EC}',
+  ['\xED'] = '\u{00ED}',
+  ['\xEE'] = '\u{00EE}',
+  ['\xEF'] = '\u{00EF}',
+  ['\xF0'] = '\u{00F0}',
+  ['\xF1'] = '\u{00F1}',
+  ['\xF2'] = '\u{00F2}',
+  ['\xF3'] = '\u{00F3}',
+  ['\xF4'] = '\u{00F4}',
+  ['\xF5'] = '\u{00F5}',
+  ['\xF6'] = '\u{00F6}',
+  ['\xF7'] = '\u{00F7}',
+  ['\xF8'] = '\u{00F8}',
+  ['\xF9'] = '\u{00F9}',
+  ['\xFA'] = '\u{00FA}',
+  ['\xFB'] = '\u{00FB}',
+  ['\xFC'] = '\u{00FC}',
+  ['\xFD'] = '\u{00FD}',
+  ['\xFE'] = '\u{00FE}',
+  ['\xFF'] = '\u{00FF}'
+}
```

Comment:
A0 to AF is in fact the identity so we could delete those entries and add that range to the range using the `utf8.char` function below (not sure which is simpler/faster)